packages/react/src/components/Card/_card.scss

```diff
@@ -31,45 +34,92 @@ $iot-header-padding: $spacing-05;
 }
 
 .#{$iot-prefix}--card--title {
-  flex: 1;
+  align-items: flex-start;
+  display: flex;
+  flex-wrap: wrap;
   // min-width value is required to ensure flex child with text properly truncates
   // https://css-tricks.com/flexbox-truncated-text/
   min-width: 0;
-  display: flex;
+  margin-top: $spacing-03;
+  padding-right: $spacing-05;
+  width: 100%;
+
+  html[dir='rtl'] & {
+    padding-left: $spacing-05;
+    padding-right: unset;
+  }
 
   & + *:not(.#{$iot-prefix}--card--toolbar) {
     margin-left: $spacing-05;
   }
 }
 
-.#{$iot-prefix}--card--title--text {
-  @include type-style('productive-heading-01');
-  display: block;
+.#{$iot-prefix}--card--header--tooltip {
+  flex-basis: $tooltipSize;
+  margin-top: $spacing-02;
+  .#{$prefix}--tooltip__trigger {
+    margin-right: $spacing-03;
+  }
+}
+
+.#{$iot-prefix}--card--title--text,
+.#{$iot-prefix}--card--subtitle--text {
   overflow: hidden;
-  text-overflow: ellipsis;
-  white-space: nowrap;
+  word-break: break-all;
+  -webkit-line-clamp: 1;
+  max-height: $lineHeightSize;
+  display: -webkit-box;
+  min-width: 3rem;
+  /* autoprefixer: ignore next */
+  -webkit-box-orient: vertical;
+}
+
+.#{$iot-prefix}--card--title--text__wrapped {
```

Comment:
You are right. I could have sworn it was flipped and `__` was the modifier. Since the class names  base is already been defined I am just removing the `__` on the two you have called out but leaving the base as is. .